backend/core/types/src/archer-listing.ts

```diff
@@ -1626,6 +1626,7 @@ export const ArcherListing: Listing = {
       amiChart,
     },
   ],
+  isExternal: false,
```

Comment:
remind me -- we do still seed with some external listings right?

Reply:
Yep.  The `isExternal` property is virtual though; it isn't stored anywhere, it's just set automatically based on which table the listing originated from.  Anything that comes from the `external_listings` table automatically has that set to true (including seed data), and there are tests on the backend that verify that 1) anything added to `external_listings` has `isExternal` set to true and 2) all listings in the `listings` table have it set to false.